coding-the-legislation/legislation_parameters.md

```diff
@@ -130,3 +130,129 @@ The [<abbr title="Institut des politiques publiques">IPP</abbr>](http://www.ipp.
 The OpenFisca team works on importing those data into the YAML parameter files of OpenFisca-France.
 
 See [this README](https://github.com/openfisca/openfisca-france/tree/master/openfisca_france/scripts/parameters/baremes_ipp) for more information.
+
+
+## Computing a parameter that depends on a variable
+
+Sometimes, the value of a paremeter depends on a variable.
```

Comment:
Sometimes, the value of a parameter depends on a variable (e.g. a housing benefit that depends on the zone the house is built on)